lte/gateway/docker/s1ap/start-s1ap.sh

```diff
@@ -10,6 +10,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Create docker networks
+docker network inspect s1&>/dev/null || {
+docker network create --driver=bridge --subnet=192.168.60.0/24 --gateway=192.168.60.142 sgi && \
+docker network create --driver=bridge --subnet=192.168.128.0/23 --gateway=192.168.129.1 s1 && \
+docker inspect s1aptester&>/dev/null && docker rm s1aptester && \
```

Comment:
There seems to be debugging code here, which should be removed. Or what is the second `docker inspect` for which should always return `true`?

Reply:
The reason these are necessary is that currently [docker-compose errors out if you specify a gateway address](https://github.com/docker/compose/issues/6569). These ip addresses are unfortunately hardcoded everywhere. 

```
root@ip-10-7-4-49:/var/opt/magma/docker# docker-compose --env-file .env -f docker-compose.yaml -f s1ap/docker-compose.s1ap.yaml config
ERROR: The Compose file './s1ap/docker-compose.s1ap.yaml' is invalid because:
networks.s1.ipam.config value Additional properties are not allowed ('gateway' was unexpected)
networks.sgi.ipam.config value Additional properties are not allowed ('gateway' was unexpected)
```

The second docker inspect is needed cause if the user exits the s1aptester container manually, and doesn't run the stop script, it errors out saying that the container already exists. This step ensures that if the container exists, it removes it.